docs/en/enterprise-edition/content-collections/application-security/risk-management/monitor-and-manage-code-build/enforcement.adoc

```diff
@@ -1,44 +1,49 @@
 == Enforcement
 
-Enforcement enables you to configure code review scan parameters in your repositories and customize violation failures and comments. On the Prisma Cloud console there are default parameters, based on best practices, for each code category scanned  in your repositories. Using enforcement  you can configure these default parameters and receive violation notifications only for critical issues, helping you reduce unnecessary noise and optimizing secure productivity.
+Prisma Cloud provides default parameters for each code category scanned in your repositories, based on best practices. Through Enforcement, you can efficiently manage enforcement strategies across your code reviews by customizing how violations, failures and comments are handled before repositories are scanned or integrated. This capability helps reduce unnecessary noise, allowing you to focus on the most critical issues.
 
+////
 Enforcement configurations scan every commit into your repository and suggest fix remedies, if any violation is detected, this is in addition to the scan that Prisma Cloud periodically performs on your repositories, the results for which are accessible on Projects.
-The periodic scans have a predefined severity threshold which are run across code categories that result in three run rules.
+////
+The scans are applied across code categories, leading to the enforcement of three run rules:
 
-* *Hard Fail*
-+
-A repository scan result fails when Prisma Cloud detects a violation or a vulnerability.
+* *Hard Fail*: A repository scan result fails when Prisma Cloud detects a violation or a vulnerability.
 
-* *Soft Fail*
-+
-A repository scan result is a pass and a notification appears on the console when Prisma Cloud detects a violation or a vulnerability.
+* *Soft Fail*: A repository scan result passes, but a notification is displayed on the console when Prisma Cloud detects a violation or a vulnerability.
 
-* *Comment bot*
-+
-A repository scan result  displays issues with fix suggestions as comments with Pull requests accessible on VCS (Version Control System).
-
-Prisma Cloud scans multiple code categories to identify vulnerabilities and violations unique to the category. See the table for more details.
+* *Comment bot*: A repository scan result displays issues with suggested fixes as comments within Pull Requests on the VCS (Version Control System). 
```

Comment:
We should be consistent with capitalization for the rule types here... suggest "Comment Bot"

Reply:
Ensured consistency by changing to "Soft fail"